src/components/TestToolMenu.js

```diff
@@ -43,7 +44,7 @@ const TestToolMenu = props => (
 
         {/* Option to switch between staging and default api endpoints.
         This enables QA and internal testers to take advantage of sandbox environments for 3rd party services like Plaid and Onfido. */}
```

Comment:
We should update the comment with additional information explaining why the "staging toggle" is conditionally rendered 
- external devs can use it during DEV
- internal devs can't use it, when they need to they change their `.env` config to use STAGING

Reply:
What you have is good, though you can say this to make it shorter:
```
This toggle is not rendered for internal employees as they make environment changes directly to the .env file.
```